metron-platform/metron-writer/src/test/java/org/apache/metron/writer/BulkWriterComponentTest.java

```diff
@@ -118,6 +118,20 @@ public void writeShouldProperlyAckTuplesInBatch() throws Exception {
   }
 
   @Test
+  public void writeShouldProperlyAckTuplesWhenWriterDisabled() throws Exception {
```

Comment:
Do you mean write a test where a writer for one sensor is enabled and a writer for another sensor is disabled?  Or do you really want separate writers?